docs/developer/references/camino-contracts.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 7
+---
+
+# Camino Smart Contracts
+
+Camino Network has multiple smart contracts to manage administrative tasks, KYC verification, gas fee voting, and dApp incentives.
```

Comment:
Multiple sounds scary, we have 1 smartcontract (also if these are 2, but  lets not count Proxy) for administrative things, and one SC for incentivation activities on the c-chain (for example d-app incentives)

Reply:
How about now? I rephrased the text on the page. :)